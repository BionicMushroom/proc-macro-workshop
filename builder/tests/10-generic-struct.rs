use derive_builder::Builder;

#[derive(Builder)]
struct Test<T, U, V> {
    field0: T,
    field1: U,
    field2: V,
}

fn main() {}
