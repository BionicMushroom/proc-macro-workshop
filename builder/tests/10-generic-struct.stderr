error: expected struct without generic type parameters
 --> tests/10-generic-struct.rs:4:12
  |
4 | struct Test<T, U, V> {
  |            ^^^^^^^^^
